moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -46,57 +47,8 @@ const useStyles = makeStyles((theme) => ({
     paddingTop: "25px",
     paddingBottom: "16px",
   },
-  colorPrimary: {
-    color: theme.palette.primary.main,
-  },
 }));
 
-/**
- * Default column display (if no config in local storage)
- */
-const DEFAULT_HIDDEN_COLS = {
-  project_id: false,
-  project_name: false,
-  current_phase: false,
-  project_team_members: true,
-  project_lead: false,
-  project_sponsor: false,
-  project_partner: true,
-  ecapris_subproject_id: false,
-  updated_at: false,
-  project_feature: true, // signal_ids
-  task_orders: true,
-  type_name: true,
-  funding_source_name: true,
-  project_note: true,
-  construction_start_date: true,
-  completion_end_date: true,
-  project_inspector: true,
-  project_designer: true,
-  contractors: true,
-  contract_numbers: true,
-  project_tags: true,
-  added_by: true,
-  public_process_status: true,
-  interim_project_id: true,
-  children_project_ids: true,
-  parent_project_id: true,
-  components: true,
-};
-
-/**
- * Keeps localStorage column config in sync with UI interactions
- * @param {Object} column - the MT column config with the `field` prop - aka the column name
- * @param {Bool} hidden - the hidden state of the column
- */
-const handleColumnChange = ({ field }, hidden) => {
-  let storedConfig =
-    JSON.parse(localStorage.getItem("mopedColumnConfig")) ??
-    DEFAULT_HIDDEN_COLS;
-  storedConfig = { ...storedConfig, [field]: hidden };
-  localStorage.setItem("mopedColumnConfig", JSON.stringify(storedConfig));
```

Comment:
This happens in `useColumns` now so it is responsible for syncing the hidden columns state with user preferences that we save in local storage.